members/admin/department_admin.py

```diff
@@ -37,7 +37,10 @@ def get_queryset(self, request):
                 "description": "<p>Lav en beskrivelse af jeres aktiviteter, teknologier og tekniske niveau.</p><p>Åbningstid er ugedag samt tidspunkt<p>",
             },
         ),
-        ("Ansvarlig", {"fields": ("responsible_name", "responsible_contact")}),
+        (
+            "Ansvarlig",
+            {"fields": ("responsible_name", "department_email", "department_leader")},
+        ),
```

Comment:
VI skal have lavet et filter på den her som nævnt her https://github.com/CodingPirates/forenings_medlemmer/pull/435#issuecomment-579315117. 

Det bedste havde været hvis vi kunne sortere på dem som er frivillig i afdelingen, men det kan ikke lade sig gøre nu. 

Hvis du stedet returnerer alle personer hvor deres user har `is_staff=True` så burde det være fint indtil vi får det. (Måske også tjekke den med `is_active`)

Se mere her https://docs.djangoproject.com/en/3.0/ref/contrib/auth/#django.contrib.auth.models.User.is_staff

Reply:
Jeg har fået lavet is_staff filtret, men kan ikke finde ud af hvordan adminuserinformation__user fungerer til at lave et filter så man kun ser folk fra ens egen department.

Så vidt jeg kan se i get_user_adress er det bare 
`department_users = Member.objects.filter(
            adminuserinformation__user=request.user
)`, men det giver bare en error med at "adminuserinformation cannot resolve into field".

Hvad er det jeg overser?

Reply:
Det var også lidt uklart skrevet af mig :) 
https://github.com/CodingPirates/forenings_medlemmer/blob/c1477de262e5bf29932ff39164e5f642b9b2b6d1/members/admin/department_admin.py#L17-L25

Hvis du ser på `get_form` koden i den her, så tager den og sætter _queryset_ (Valgmuligheder) til den liste af adresser vi får fra `Address.get_user_address()`

I stedet for den der `Address.get_user_address` skal vi have en liste af personer som kan blive afdelingsledere. 

Hvis du starter med at sige noget i stil med 
```python 
form.base_fields['department_leader'] = Person.objects.filter("person__user.isstaff")
```
Så får vi kun den liste af personer der har admin adgang

Reply:
Filter implementeret i nyeste commit, brugte en lidt anden fremgangsmåde en get_form, men hvis det giver problemer kan det vist nemt overføres dertil.